common/include/Utilities/PString.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+#include "ghc/filesystem.h"
+#include <iostream>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <locale.h>
+#include <string>
+#include "Path.h"
+
+namespace fs = ghc::filesystem;
```

Comment:
This will make every file including `PString.h` use this alias, not a good idea. Probably better to move this alias to inside the class.